lib/flipper/adapters/strict.rb

```diff
@@ -0,0 +1,44 @@
+module Flipper
+  module Adapters
+    # An adapter that ensures a feature exists before checking it.
+    class Strict
+      extend Forwardable
+      include ::Flipper::Adapter
+      attr_reader :name, :adapter
+
+      class NotFound < ::Flipper::Error
+        def initialize(name)
+          super "Could not find feature #{name.inspect}. Call `Flipper.add(#{name.inspect})` to create it."
+        end
+      end
+
+      HANDLERS = {
+        raise: ->(feature) { raise NotFound.new(feature.key) },
+        warn: ->(feature) { warn NotFound.new(feature.key).message },
+        noop: ->(_) { },
+      }
+
+      DEFAULT_NOT_FOUND =
```

Comment:
Is this magic or missing?

Reply:
Doh, that is unused now. Funny that it wasn't raising an error.